### PR TITLE
Add state totals by candidate to the state heading

### PR DESCRIPTION
**Pull request by fractionalhare**
Resolves issue #54. Adds state vote totals for each candidate to the .txt and .html files in the state heading.

### CODE REVIEW

File: print-battleground-state-changes [hunk 1]
```diff
@@ -275,14 +281,14 @@ print(tabulate([
...
-    print(f'\n{state}:')
+    print(f'\n{state}: Total Votes: ({timestamped_results[1][1]}: {timestamped_results[1][3]}, {timestamped_results[1][2]}: {timestamped_results[1][4]})')
     print(tabulate([string_summary(summary) for summary in timestamped_results]))
 
     # 'Alaska (3)' -> 'alaska', 'North Carolina (15)' -> 'north-carolina'
     state_slug = state.split('(')[0].strip().replace(' ', '-').lower()
     html_chunks.append(f"<table id='{state_slug}' class='table table-bordered'>")
-    html_chunks.append(html_write_state_head(state))
     for summary in timestamped_results:
+        html_chunks.append(html_write_state_head(state, summary))
```
Comment (alex) [inline print-battleground-state-changes]: Writing the thead for every row seems like a lot? Can you post a screenshot of what this looks like

Comment (fractionalhare) [reply print-battleground-state-changes]: Good spot! It looks wonky, let me refactor that :)

Comment (fractionalhare) [reply print-battleground-state-changes]: done. and this is what it looks like now. Screen Shot 2020-11-05 at 7 35 43 PM

Comment (fractionalhare) [reply print-battleground-state-changes]: should I also make another commit emphasizing that the parenthetical is for EV?

**Pull request merged**